docker/go-control-plane/README.md

```diff
@@ -14,3 +14,12 @@ the root of `grpc/psm-interop` checkout.
 ```
 docker build . -f docker/go-control-plane/Dockerfile
 ```
+
+## Local development
+
+Run the following command from this repository root to generate protocol files:
```

Comment:
You are not generating protocol files here. I would say something like:
`Run the following command from this repository to compile the .proto files` or
`Run the following command from this repository to generate code from the .proto files`.

Reply:
Done